e2e/nomostest/sync.go

```diff
@@ -257,35 +257,46 @@ func statusHasSyncDirAndNoErrors(status v1beta1.Status, sourceType v1beta1.Sourc
 	if status.Source.ErrorSummary != nil && status.Source.ErrorSummary.TotalCount > 0 {
 		return fmt.Errorf("status.source contains %d errors:\n%s", status.Source.ErrorSummary.TotalCount, objJSON)
 	}
-	if sourceType == v1beta1.OciSource {
-		if ociDir := status.Source.Oci.Dir; ociDir != dir {
-			return fmt.Errorf("status.source.ociStatus.dir %q does not match the provided directory %q:\n%s", ociDir, dir, objJSON)
-		}
-	} else if gitDir := status.Source.Git.Dir; gitDir != dir {
-		return fmt.Errorf("status.source.gitStatus.dir %q does not match the provided directory %q:\n%s", gitDir, dir, objJSON)
-	}
 	if status.Sync.ErrorSummary != nil && status.Sync.ErrorSummary.TotalCount > 0 {
 		return fmt.Errorf("status.sync contains %d errors:\n%s", status.Sync.ErrorSummary.TotalCount, objJSON)
 	}
-	if sourceType == v1beta1.OciSource {
-		if ociDir := status.Sync.Oci.Dir; ociDir != dir {
-			return fmt.Errorf("status.sync.ociStatus.dir %q does not match the provided directory %q:\n%s", ociDir, dir, objJSON)
-		}
-	} else if gitDir := status.Sync.Git.Dir; gitDir != dir {
-		return fmt.Errorf("status.sync.gitStatus.dir %q does not match the provided directory %q:\n%s", gitDir, dir, objJSON)
+	if message := status.Rendering.Message; message != parse.RenderingSucceeded && message != parse.RenderingSkipped {
+		return fmt.Errorf("status.rendering.message %q does not indicate a successful state:\n%s", message, objJSON)
```

Comment:
This if statement was originally after the next line that checks rendering error - is it still correct to move it before? It seems that rendering could be either in progress or failed, moving this changes the original behavior.

Reply:
Done